readme.txt

```diff
@@ -61,6 +61,9 @@ Astra Bulk Edit plugin can be used only with the Astra theme.
 
 == Changelog ==
 
+= 1.2.5 =
+- Improvement: Hardened the security of plugin
```

Comment:
@imnavanath Please add a full stop '.' at the end.

Reply:
Done 👍